Test/comp/CSharpStyling2.cs

```diff
@@ -1,37 +1,49 @@
 using System.Numerics;
 
-namespace _module {
-  public partial class MyClass {
-    public MyClass() { }  // needed by Dafny in order to call constructor "Init"
-    public MyClass(BigInteger x) {
-      this.u = x;
-    }
-    public MyClass(bool y) {
-      this.u = y ? 3 : -3;
-    }
+namespace _module
+{
+    public partial class MyClass
+    {
+        public MyClass() { }  // needed by Dafny in order to call constructor "Init"
```

Comment:
All of these `Test/comp/*.cs` files have a different formatting. For example, the (new) code uses 4 spaces for indent and places many `{` on the next line for an `if`. Do the same code-style rules perhaps not apply in the same way for the `Test/comp` directory?

Reply:
Very good point. No, they didn't, only the files in `Source` were affected by `.editorconfig`. I moved it and reran `dotnet-format`. We don't check files beyond the scope of `Source/Dafny.sln` in the CI. Should we?